### ISSUE TITLE
TOR proxy setting don't work right after Orbot installation

### ISSUE THREAD

**Issue by karina-archazh** [AdguardTeam/AdguardForAndroid#2167]
1. On Proxy settings choose Tor integration.
2. Download Orbot from store.
3. Return to Adguard settings and push integration button.
Nothing happens.

Integration with Orbot start work only after Orbot' first launch.

**Comment by zebrum** [comment]
Basically we need to show a notification about that to users.

**Comment by ameshkov** [comment]
What we really should do is:

1. Show progress bar right after the first tap on this button
2. If there is a timeout error, show alert with the explanation what could be causing this issue, and how to fix it.

**Comment by RaXorX** [comment]
I am on nightly adguard at the moment. v4.0 nightly 32 (73)
I when I clicked on the integrate with tor button it didn't show the play store page to me (if it is supposed to show to install). Even after I installed the orbot app, the integrate with tor button does nothing and I had to manually set up the proxy for it.
To have tor working I have to manually open the app myself. So disabling and enabling adguard toggle/filtering does nothing at all to start it up. If I have to enable or disable tor I have to manually do so from the orbot app.

https://github.com/AdguardTeam/AdguardForAndroid/issues/3220
This here is an issue for Orbot not stopping when adguard protection is turned off. In my case it doesn't even start nor stop.
